posthog/session_recordings/queries/test/test_session_recording_list_from_session_replay.py

```diff
@@ -1,3115 +0,0 @@
-from datetime import datetime
-from uuid import uuid4
-
-from dateutil.relativedelta import relativedelta
-from django.utils.timezone import now
-from freezegun import freeze_time
-
-from posthog.clickhouse.client import sync_execute
-from posthog.clickhouse.log_entries import TRUNCATE_LOG_ENTRIES_TABLE_SQL
```

Comment:
Probably still need this because it's used in the `test_logger` file but the only other references were the two deleted test files in this PR